password-hash/src/errors.rs

```diff
@@ -41,14 +52,11 @@ pub enum Error {
     /// Invalid password.
     Password,
 
-    /// Password hash string contains invalid characters.
-    PhcStringInvalid,
-
-    /// Password hash string too short.
-    PhcStringTooShort,
+    /// Password hash string invalid.
+    PhcStringField,
 
-    /// Password hash string too long.
-    PhcStringTooLong,
+    /// Password hash string contains leftover characters.
+    PhcStringLeftover,
```

Comment:
I think this phrase has precedence in this situation:

```suggestion
    /// Password hash string contains trailing data.
    PhcStringTrailingData
```